site/content/blog/kctrl-release-blog.md

```diff
@@ -0,0 +1,58 @@
+
+## How does `kctrl` help package consumers?
+Traditionally, installing a package would involve setting up relevant cluster roles and service accounts which allow the package installations to create desired resources on the cluster. In case the user is configuring the package, they would need to create a Secret which makes this information available and then reference it in their PackageInstall resource. These resources had to be authored and tracked by the user.
+
+`kctrl` helps users get up and running with a pacakage installation with as little as
+```bash
+$ kctrl package install -i cert-man -p cert-manager.community.tanzu.vmware.com --version 1.6.1 --values-file values.yaml
+```
+Where, `values.yaml` defines configuration for the installation.
+
+`kctrl` creates sevice accounts and role bindings with required permissions and keeps track of them. If custom configuration is provided by the user, `kctrl` creates a secret and adds required references to the custom resources being created. When a package is uninstalled, these resources are garbage collected and deleted from the cluster as well.
+
+In case, the user has to use a service account with limited permissions, the same can be supplied to the CLI while installing the application. In this case, kapp-controller does not try to create and track the required resources.
+
+`kctrl` also allows users to add PacakgeRepositories published as `imgpkg` bundles easily to their clusters.
+```bash
+$ kctrl package repository add -r tce --url projects.registry.vmware.com/tce/main:0.10.0
+```
+
+You can find a more detailed end to end workflow for consuming published packages, configuring them and observing them over [here](/kapp-controller/docs/develop/kctrl-package-tutorial/)!
+
+## What is next for `kctrl`?
+We want to help our users to easily observe what kapp-controller is doing. `kctrl` aims to do this by surfacing information about what kapp-controller does with the configuration provided by the user in a transparent, legible and usable manner.
+
+This enables developers authoring App CRs and Carvel packages to iterate faster by observing how their configuration affects the cluster. It also enables package consumers to narrow down on the root cause of erroneous installations more conveniently.
+
+Some of the upcoming improvements aimed at demystifying what goes on under the hood and improving the debugging workflow are:
+- Observe and follow what an App CR does on the cluster. `kctrl` will surface information revealed by an AppCRs status fields in a readable and sequential manner.
+- Observe and inspect resources created by App CRs easily.
+- Allow users to pause and trigger reconciliations for App CRs and PackageInstalls easily. This allows users to debug these resources more efficiently.
+- The benefits of enhanced observability of App CRs will also extended to AppCRs created due to PackageInstalls.
```

Comment:
nit: `by` instead of `due to`

Reply:
Not sure about this as the resource PackageInstall does not create the App CR, but the controller does when it notices a PackageInstall. Leaving this open for discussion